docs/tutorial/submit.md

```diff
@@ -18,7 +18,7 @@ spark-client.spark-submit \
 local:///opt/spark/examples/jars/$SPARK_EXAMPLES_JAR_NAME 100
 ```
 
-#### Adding Big Data to the mix
+### Adding Big Data to the mix
 
 Once the setup is validated, it's time to test out with a real big data workload. Here we assume that 
 * the input data is placed in S3
```

Comment:
this bit is a bit magical. We need to have a setup guide for object storage referenced here

Reply:
Yes, indeed. I agree. I'll place an item in the backlog for Abhi for updating documentation with this

Reply:
https://warthogs.atlassian.net/browse/DPE-1574